### DISCUSSION TITLE
flaiR installation error

### DISCUSSION THREAD

**Discussion by Sarangismruti** [davidycliao/flaiR#123] [General]
I tried to install flair in Windows 11 and this error came  out: 

> library(flaiR)
flaiR: An R Wrapper for Accessing Flair NLP Traceback (most recent call last):
Warning message:
In system(flair_version_command, intern = TRUE) :
  running command 'C:\Users\SMRUTI~1\AppData\Local\Programs\Python\PYTHON~1\python.exe -c "import flair; print(flair.__version__)"' had status 1

**Comment by paula-montano** [comment]
Hi Smruti, 

To fix this issue, you can:
1. install Microsoft C++ https://visualstudio.microsoft.com/visual-cpp-build-tools/
2. uninstall all the previous versions of Python and Anaconda on your PC.
3. Reinstall Python 3.12.3 at https://www.python.org/downloads/ and tick the two boxes in the installation window, including Adding python.exe to PATH. 
4. Finally, you can reinstall flaiR in RStudio. If you have previously installed flaiR, make sure you remove it, restart your session, and install it again.
